cmd/cvdr/main.go

```diff
@@ -24,24 +24,31 @@ import (
 	"github.com/google/cloud-android-orchestration/pkg/client"
 )
 
-const configPathVar = "CVDR_CONFIG_PATH"
+const (
+	envVarConfigPathVar     = "CVDR_CONFIG_PATH"
```

Comment:
This is not necessary, use the user and default configs only and allow having any combination of them

Reply:
Done.